CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
-# 4.4.1 (Unreleased)
+# 5.0.0 (Unreleased)
```

Comment:
Since we are supporting a new API version, the suggestion from team was to release it as a new version. @soodpr any comments here?

Reply:
Sure I understand, however the project is currently using [Semantic Versioning](https://semver.org/) as stated [here](https://github.com/HewlettPackard/python-hpOneView/pull/343/files/7bfcea9443f19d0c8605cedc681c7eb6d302b657#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR58), and SemVer has its own rules regarding release numbers. 
While I understand wanting to bump up the major version for each new API released, during the development cycle for a new API a new breaking change will *probably* end up being introduced anyway, but we should likely strive to avoid those.